go.mod

```diff
@@ -65,19 +77,27 @@ require (
 	github.com/syndtr/goleveldb v1.0.1-0.20190625010220-02440ea7a285
 	github.com/tedsuo/ifrit v0.0.0-20180802180643-bea94bb476cc
 	github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8 // indirect
+	github.com/vektra/mockery v0.0.0-20181123154057-e78b021dcbb5 // indirect
 	github.com/willf/bitset v1.1.10
 	github.com/xordataexchange/crypt v0.0.3-0.20170626215501-b2862e3d0a77 // indirect
 	go.etcd.io/etcd v0.5.0-alpha.5.0.20181228115726-23731bf9ba55
-	go.uber.org/zap v1.14.0
+	go.uber.org/atomic v1.4.0 // indirect
+	go.uber.org/multierr v1.2.0 // indirect
+	go.uber.org/tools v0.0.0-20190618225709-2cfd321de3ee // indirect
+	go.uber.org/zap v1.11.0
```

Comment:
especially weird that this would go backwards.

Reply:
ah, you did the tools first. I can slap the third commit for the verify on top of those two and give it a go.

I'm worried about what happens in a clean environment in terms of it changing go.mod when building the tools.

Reply:
I had a CRLF git issue with 
`vendor/github.com/Knetic/govaluate/.gitignore`
but otherwise got it cleanly building.

Looks to work in the build as well, besides some gossip flaknes: https://dev.azure.com/mbauer0254/fabric/_build/results?buildId=4&view=results

I'll push a modified rebase with everything, and you can check it out or modify it yourself on my fork, or push your own. Let me know what you want to do.